README.MD

```diff
@@ -1,6 +1,12 @@
 BrandedToken.js
 ============
 
+![Build master](https://img.shields.io/travis/OpenSTFoundation/brandedtoken.js/master.svg?label=build%20master&style=flat)
+![Build develop](https://img.shields.io/travis/OpenSTFoundation/brandedtoken.js/develop.svg?label=build%20develop&style=flat)
+![npm version](https://img.shields.io/npm/v/@openstfoundation/mosaic.js.svg?style=flat)
```

Comment:
`brandedtoken.js`, not `mosaic.js` 😉

Reply:
Thank you for pointing out 💯. 
Updated!